react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+type topics = {
+  topic: string;
+  id: number;
+  information: string;
+};
+
+type Props = {
+  topicArray: topics[];
+};
+
+export default function Accordion({ topicArray }: Props) {
+  return (
+    <div>
+      <Topics topicArray={topicArray} />
+    </div>
+  );
+}
+
+function Topics({ topicArray }: Props) {
+  const [clickedTopic, setClickedTopic] = useState('');
```

Comment:
Instead of a string, you could make `clickedTopic` be an actual Topic. This would also make it work correctly in the case where two topics have the same title.